docs/src/Datapoint.doc.js

```diff
@@ -0,0 +1,140 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Datapoint } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+import FeedbackCallout from './components/FeedbackCallout.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(<FeedbackCallout componentName="Datapoint" />);
+
+card(
+  <PageHeader
+    name="Datapoint"
+    description="Datapoint displays at-a-glance data for a user to quickly view key metrics."
+    defaultCode={`
+<Datapoint size="lg" infoText="Contextual information for this Datapoint" title="Datapoint title" value="1.23M" trend={{value: 30, accessibilityLabel: "Trending up"}} helperText="Datapoint helper text" />
+`}
+    pilot
+  />,
+);
+
+// Specifying Component gives auto warnings about missing props
+card(
+  <PropTable
+    Component={Datapoint}
+    props={[
+      {
+        name: 'title',
+        type: 'string',
+        description: 'The header for the component',
+        required: true,
+      },
+      {
+        name: 'value',
+        type: 'string',
+        description: 'The main datapoint value (e.g., 1.23M)',
+        required: true,
+      },
+      {
+        name: 'trend',
+        type: '{| accessibilityLabel: string, value: string |}',
+        description: `Object detailing the trend value (change in time - e.g., +30%), and accessibilityLabel to describe the trend's icon (e.g., "Trending up")`,
+      },
+      {
+        name: 'trendSignal',
+        type: `'good' | 'bad' | 'neutral' | 'auto'`,
+        defaultValue: 'auto',
+        description: `A visual indicator whether the trend is considered "good", "bad" or "neutral". By setting \`trendSignal\` to \`auto\`, a positive trend will be considered "good", a negative trend will be considered "bad" and a trend of zero will be considered "neutral"`,
+      },
+      {
+        name: 'infoText',
+        type: 'string',
+        description: 'Contextual information displayed in a tooltip to describe the Datapoint',
+      },
+      {
+        name: 'size',
+        type: `'md' | 'lg'`,
+        defaultValue: 'md',
+        description: 'Used to set the size of the datapoint',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`title\`, \`infoText\` and \`trendAccessibilityLabel\` props. Note that localization can lengthen text by 20 to 30 percent.`}
+  />,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection columns={2} title="Size">
+      <MainSection.Card
```

Comment:
These should use real titles as well (or at least real-ish)

Reply:
Good callout. Done.